sashimi-webapp/src/components/file-manager/UserInputs.vue

```diff
@@ -118,6 +132,13 @@ export default {
       }
     },
     setViewMode(viewMode) {
```

Comment:
One vue-data `viewMode` is sufficient for holding this information. `iconViewMode` and `listViewMode` is not necessary. As the CSS can make use of the `viewMode` directly. e.g. 

```
div[viewMode="iconMode"] {
}
div[viewMode="listMode"] {
}
```

Using this approach, we can easily expand additional viewMode easily with just CSS stylesheet